Makefile

```diff
@@ -92,10 +92,7 @@ stop: ## Stops webpack
 ifeq ($(OS),Windows_NT)
 	wmic process where "Caption='node.exe' and CommandLine like '%webpack%'" call terminate
 else
-	@for PROCID in $$(ps -ef | grep "[n]ode.*[w]ebpack" | awk '{ print $$2 }'); do \
-		echo stopping webpack watch $$PROCID; \
-		kill $$PROCID; \
-	done
+	@pkill -fl webpack || true
```

Comment:
The new version of webpack shows up as `webpack` when you run `ps -ef` instead of `node /some/path/to/webpack`, so the old code didn't work. Luckily, MacOS has come with `pkill` for a couple years now, so we can finally use that everywhere.

The `|| true` is needed since `pkill` returns an error code when no processes are found to kill.